assets/style.css

```diff
@@ -168,24 +166,18 @@ footer {
 .card {
   display: flex;
   flex-direction: column;
-  /* border: 1px solid var(--dark-purple); */
+  border: 1px solid var(--dark-purple);
```

Comment:
Since this was commented out, we are no longer using it, You can also delete this